### DISCUSSION TITLE
Коммерческое использование конфигурации

### DISCUSSION THREAD

**Discussion by 1c-spec** [Polyplastic/1c-parsing-tech-log#92] [General]
Добрый день!
Уточните пожалуйста, есть ли ограничения/условия на использование конфигурации 1c-parsing-tech-log в коммерческих целях?

**Comment by ivanov660** [comment]
п.1. Конфигурация бесплатная и платной делать не планируем. Использовать можете как в личных так и в коммерческих целях, соблюдайте п.2 и учитывайте п.3.
п.2. Авторское право. Сохраняйте ссылку и информацию на репозиторий и авторов.
п.3. Отказ от гарантий. Мы не предоставляем какой-либо гарантии по использованию этого ПО или возмещения какого-либо прямого или косвенного возможного ущерба и т.п. (программа предоставляется «как есть»).

**Reply by lo0p3r** [reply]
Предлагаю добавить в репозиторий файл LICENSE с текстом [MIT-лицензии](https://en.wikipedia.org/wiki/MIT_License)
